.expeditor/verify.pipeline.yml

```diff
@@ -37,13 +37,6 @@ steps:
   expeditor:
     executor:
       docker:
-- label: rspec 2.3
-  command:
-    - asdf install ruby 2.3.8
-    - asdf local ruby 2.3.8
-    - cd /workdir/components/ruby
-    - bundle install
-    - bundle exec rspec
   timeout_in_minutes: 15
```

Comment:
We can get rid of the following lines here too. Everything else runs in under 5 minutes